configure.ac

```diff
@@ -1086,6 +1086,9 @@ AM_CONDITIONAL([ENABLE_DEVELOPMENT], [test -n "$MET_DEVELOPMENT"])
 
 CPPFLAGS=$CPPFLAGS' -DMET_BASE="\"$(pkgdatadir)\""'
 
+# Add -std=c++11 to CXXFLAGS                                                                                                                                                 
+CXXFLAGS=$CXXFLAGS' -std=c++11'
```

Comment:
Adding the `-std` flag to `configure.ac` makes me a bit nervous. I worry about whether any of the compilers out there will balk at that option and complain that it's not supported.